### PR TITLE
Running page

### PR DESCRIPTION
**Pull request by MasayukiNagai**

### CODE REVIEW

File: src/assets/styles/firstpage.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  border-radius: 50px;
+  cursor: pointer;
+  display: inline-block;
+  color: white;
+  padding: 20px 24px;
+  background-color: rgb(255, 100, 100);
+  text-decoration: none;
+}
+
+/*#target_point_span{
```
Comment (sonegishi) [inline src/assets/styles/firstpage.css]: Delete commented lines

File: src/assets/styles/firstpage.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+@keyframes background-color {
+  0% { color: #F9136B; }
+  20% { color: #f1c40f; }
+  40% { color: #1abc9c; }
+  60% { color: #3498db; }
+  80% { color: #9b59b6; }
+  100% { color: #e74c3c; }
+}*/
```
Comment (sonegishi) [inline src/assets/styles/firstpage.css]:
```suggestion
}*/

```

File: src/App.vue [hunk 3]
```diff
@@ -23,6 +21,5 @@ export default {
   margin-top: 60px;
```
Comment (sonegishi) [inline src/App.vue]:
```suggestion
```

**Review by sonegishi — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 4]
```diff
@@ -1,69 +1,21 @@
...
-          </div>
-
-          <div class="submit-button">
-            <input type="submit" value="Submit" class="info-submit">
-          </div>
-
-      </div>
-          <!-- <div id="app"></div> -->
+      <h2>Trip Match</h2>
+        <span id="target_point_span"><a href="#target_point" class="target_point">Let's Search!!</a></span>
```
Comment (sonegishi) [inline public/index.html]: Make sure to change line on `a`.

File: public/index.html [hunk 5]
```diff
@@ -1,69 +1,21 @@
...
     </div>
   </div>
-
 </body>
-<footer>
-</footer>
-</html>
+</html>
```
Comment (sonegishi) [inline public/index.html]:
```suggestion
</html>

```

File: public/main_form.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```
Comment (sonegishi) [inline public/main_form.html]: Delete this file

File: src/assets/styles/firstpage.css [hunk 7]
```diff
@@ -0,0 +1,41 @@
+* {
+	margin: 0;
```
Comment (sonegishi) [inline src/assets/styles/firstpage.css]: indent

File: src/assets/styles/firstpage.css [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+body {
+  height: 800px;
+  background-image: url("../top.jpg");
+  background-size: cover;
+}
+
+.head_title {
+	text-align: center;
```
Comment (sonegishi) [inline src/assets/styles/firstpage.css]: indent

File: src/components/Select.vue [hunk 9]
```diff
@@ -7,10 +7,10 @@ export default {
 		props: {
```
Comment (sonegishi) [inline src/components/Select.vue]: Delete this file

File: src/assets/styles/button.css [hunk 10]
```diff
@@ -1,45 +0,0 @@
-.content {
```
Comment (sonegishi) [inline src/assets/styles/button.css]: Cancel this changes made

File: src/App.vue [hunk 11]
```diff
@@ -20,9 +18,7 @@ export default {
   -moz-osx-font-smoothing: grayscale;
   text-align: center;
   color: #2c3e50;
-  margin-top: 60px;
 }
 
-@import './assets/styles/button.css';
```
Comment (sonegishi) [inline src/App.vue]: Cancel

File: src/App.vue [hunk 12]
```diff
@@ -20,9 +18,7 @@ export default {
...
-@import './assets/styles/text.css';
```
Comment (sonegishi) [inline src/App.vue]: Cancel

**Review by sonegishi — CHANGES_REQUESTED** [review]

File: src/assets/styles/firstpage.css [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+.head_title h2 {
+  font-family: 'Coiny', cursive;
+  font-size: 90px;
+  color: #ffa7b6;
+  padding: 50px;
+  transition: all 0.5s;
+}
+
+.head_title h2:hover {
+  color: rgb(250, 50, 120)
```
Comment (keiohtani) [inline src/assets/styles/firstpage.css]: don't forget ; at the end.

**Review by keiohtani — CHANGES_REQUESTED** [review]

**Pull request closed without merging**